src/language/custom-types/members.md

```diff
@@ -329,3 +329,22 @@ impl Point {
     pub fn set_y(self, val: i32) -> Self { Self::new(self.x, val) }
 }
 ```
+
+It is also possible to use the struct update syntax to achieve something similar. This moves the original value:
```

Comment:
I'd suggest to reword this to avoid saying it can achieve something similar. I'd instead say it _appears_ to offer something similar, but is actually different and how:

```suggestion
Rust also has a _[struct update syntax]_ that may seem similar to `with` in
C#, but it has a very different behaviour, works fields only and _moves_ the
original value:
```